src/main/webapp/app/exercises/shared/result/result.utils.ts

```diff
@@ -283,6 +291,13 @@ export const getResultIconClass = (result: Result | undefined, templateStatus: R
         return faQuestionCircle;
     }
 
+    if (resultIsPreliminary(result) && result.participation?.exercise?.type === ExerciseType.TEXT) {
```

Comment:
What about the ai feedback for programming exercises? The UI should be the same for both...